addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -213,7 +213,7 @@ img.padding-xl, .img.padding-xl, span.fa.padding-xl, iframe.padding-xl {
 img.ms-auto, img.mx-auto {
     display: block;
 }
-
+// TODO: adapt the following rules only if we update the BS rules.
 .fa-6x {
     font-size: 6em;
 }
```

Comment:
I think it does not make sense to make the font size of fa icons responsive only from 6x. We should make them all or none at all.